File: src/SIMPLEPROGRAMS/AverageOfOddNumbersBelowARange/AverageOfOddNumbersBelowARange.java
package SIMPLEPROGRAMS.AverageOfOddNumbersBelowARange;

import java.util.Scanner;
import java.lang.*;

class AverageOfOddNumbersBelowARange{

    int value = 0,count = 0;
    int process(int limt){
        for(int i=1;i<=limt;i++){
            if((i%2)!=0){
                this.value+=i;
                this.count+=1;
            }
        }
        return this.value/this.count;
    }
    public static void main(String []args){
        Scanner io = new Scanner(System.in);
        System.out.print("ENTER THE LIMT TO FIND AVG ODD : ");
        int limt= io.nextInt();
        AverageOfOddNumbersBelowARange key = new AverageOfOddNumbersBelowARange();

        System.out.println("THE VALUE IS : "+key.process(limt));
    }
}

File: src/SIMPLEPROGRAMS/AverageOfOddNumbersBelowARange/AverageOfOddNumbersBelowARange.py
from multiprocessing.sharedctypes import Value


class oddavg:

    def __init__(self):

        global value,count

        limt = int(input("ENTER THE LIMIT :"))

        result=self.process(limt)

        print("THE VALUE IS  :",result)
    def process(self,l):
        self.l=l
        value=0
        count=0
        for i in range(0,l+1,1):
            if((i%2)!=0):
                value+=i
                count+=1
        return value/count;        
key=oddavg()